2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -2,78 +2,190 @@
+
+- avere una migliore comprensione dello stato del magazzino;
+- disporre le operazioni logistiche con maggiore cognizione.
+
+== Panoramica del prodotto
+
+=== Interazioni
+
+WMS3 si integra con, ma non comprende nel proprio ambito:
++ database SQL esterno per ottenere lo stato interno del magazzino;
++ sistema esterno per la notifica degli ordini di movimentazione tramite API RESTful.
+
+// Qui ci starebbe un bel diagramma...
+
+==== Interfacce utente
+
+WMS3 è una _web application_ acceduta e operata tramite browser. L'interfaccia utente (IU) è _web-based_ e _responsive_.
+
+Lo scenario di interazione primario avviene tramite mouse e tastiera. Tuttavia, è prevista la piena operabilità anche tramite touchscreen. Sarà possibile operare da dispositivi mobili quali tablet e smartphone.
+
+Le funzionalità esposte all'utente variano in base all'ampiezza della _viewport_ del dispositivo in uso.
+
+==== Interfacce hardware
+
+Il prodotto è acceduto tramite browser. Deve supportare l'esecuzione sui seguenti dispositivi:
+- computer desktop, tramite mouse e tastiera;
+- tablet, tramite touchscreen;
+- smartphone, tramite touchscreen.
+
+Il browser e il dispositivo devono essere compatibili con lo standard WebGL.
+
+Il prodotto non prevede elementi hardware propri o interfacce con elementi hardware di terze parti.
+
+==== Interfacce software
+
+WMS3 richiede l'accesso in lettura ad un database SQL per il caricamento e la visualizzazione dei dati.
 
-+ *Tracciabilità*: per ogni requisito si deve indicare la fonte da dove è stato ricavato, in modo da semplificare il processo di verifica della completezza e correttezza.
+==== Interfacce di comunicazione
 
-== Riferimenti
-=== Normativi
+Per la comunicazione tra le sue componenti, con l'utente e con servizi esterni, WMS3 utilizza HTTP.
 
-- Norme di progetto;
+==== Vincoli di memoria
 
+Non sono definiti vincoli o limiti sulle memorie primaria e secondaria.
+
+==== Operazioni
+
+// Dettagliare le operazioni, una volta confermate, secondo quanto descritto da 9.6.4.7
+
+==== Requisiti di adattamento al contesto
+
+// Descrivere le modalità di adattamento a diversi DBMS SQL come da 9.6.4.8
+
+==== Interfacce a servizi
+
+WMS3 dovrà invierà messaggi ad uno o più servizi esterni per comunicare gli ordini di movimentazione richiesti dall'utente. Dovrà inoltre ricevere e gestire messaggi che comunicano l'esito dell'ordine di movimentazione richiesto.
+
+=== Funzionalità del prodotto
+
+// TODO: 9.6.5
+
+=== Caratteristiche degli utenti
+
+L'utente tipico di WMS3 è un supervisore di magazzino. Ci si aspetta che la maggior parte degli accessi a WMS3 avvengano da ufficio, tramite un computer desktop dotato di mouse e tastiera. Tuttavia, non si può escludere che l'utente possa accedere a WMS3 tramite dispositivo mobile.
```

Comment:
a me continua a non piacere il Tuttavia come inizio frase come prima. Lascio la scelta a te, te lo segnalo solo come spunto di discussione

Reply:
Risolto in maniera democristiana